gridstatus/decorators.py

```diff
@@ -387,3 +390,13 @@ def __ladd__(self, other):
 
     def __radd__(self, other):
         return self.__ladd__(other)
+
+class FiveMinOffset:
+    def __ladd__(self, other):
+        # round up to next 5 min interval
+        # add 1 microsecond to ensure we make it to the 
+        # next interval when already on a 5 min interval
+        return (other + pd.Timedelta(microseconds=1)).ceil("5min")
```

Comment:
@WillKoehrsen to use ceil i needed to add 1 micro second since 

```python
>>> pd.Timestamp('2024-01-14 12:05:00').ceil("5Min")
Timestamp('2024-01-14 12:05:00')
```